base/gfspapp/task_options.go

```diff
@@ -250,6 +275,8 @@ func (g *GfSpBaseApp) TaskPriority(task coretask.Task) coretask.TPriority {
 		return coretask.UnSchedulingPriority
 	case coretask.TypeTaskGCMeta:
 		return coretask.UnSchedulingPriority
+	case coretask.TypeTaskRecoverPiece:
+		return coretask.DefaultLargerTaskPriority
```

Comment:
recovery piece task maybe use smaller task priority is better?

Reply:
changed to  smaller task priority